backend/apps/pr_review_bot/services/github_client.py

```
@@ -0,0 +1,82 @@
+        self.headers = {
+            'Authorization': f'token {self.token}',
+            'Accept': 'application/vnd.github.v3+json'
+        }
+        self.base_url = 'https://api.github.com'
+
+    def get_pr_files(self, repo: str, pr_number: int) -> List[Dict]:
+        """
+        Get files changed in PR.
+        """
+        url = f"{self.base_url}/repos/{repo}/pulls/{pr_number}/files"
+        response = requests.get(url, headers=self.headers)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)